fern/api/definition/transaction.yml

```diff
@@ -0,0 +1,173 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/fern-api/fern/main/fern.schema.json
+
+imports:
+  commons: commons.yml
+
+types:
+  Transaction:
+    union:
+      reorderPackages: ReorderPackagesTransaction
+
+      # package-specific transactions
+      createPackage: CreatePackageTransaction
+      renamePackage: RenamePackageTransaction
+      deletePackage: DeletePackageTransaction
+      reorderEndpoints: ReorderEndpointsTransaction
+      reorderTypes: ReorderTypesTransaction
+      reorderErrors: ReorderErrorsTransaction
+
+      # endpoint-specific transactions
+      createEndpoint: CreateEndpointTransaction
+      renameEndpoint: RenameEndpointTransaction
+      deleteEndpoint: DeleteEndpointTransaction
+
+      # type-specific transactions
+      createType: CreateTypeTransaction
+      renameType: RenameTypeTransaction
+      deleteType: DeleteTypeTransaction
+
+      # error-specific transactions
+      createError: CreateErrorTransaction
+      renameError: RenameErrorTransaction
+      deleteError: DeleteErrorTransaction
```

Comment:
looks great -- are you thinking `updateTransactions` come later ?

Reply:
yes